oeps/oep-templates/pep-based-template.rst

```diff
@@ -45,10 +45,10 @@ this Open edX proposal (OEP) addresses.
 Motivation
 ==========
 
-The motivation is critical for OEPs that will change Open edX. Explain why the
-existing architecture or process is inadequate to address the problem that the
-OEP solves, or why adopting the best practice would significantly improve Open
-edX.
+The motivation is critical for OEPs that will change any part of the Open edX
+ecosystem. Explain why the existing architecture or process is inadequate to
+address the problem that the OEP solves, or why adopting the best practice
+would significantly improve the Open edX world.
```

Comment:
Is there a new recommended usage of `Open edX` that we should be aware of?

Reply:
As a trademark, "Open edX" should always be used as an adjective to modify some noun that we produce.  We and legal are just getting more disciplined about this. I should explain this to everyone somehow.